packages/dds/merge-tree/src/localReference.ts

```diff
@@ -17,6 +17,18 @@ import {
 	refTypeIncludesFlag,
 } from "./referencePositions";
 
+export enum SlidingPreference {
+	/**
+	 * Prefer sliding towards the start of the string
```

Comment:
```suggestion
	 * Prefer sliding towards the end of the tree
```
and similarly below. (these are mixed up, right?)